src/order/filter/basic.lean

```diff
@@ -2050,6 +2054,10 @@ lemma tendsto_infi' {f : α → β} {x : ι → filter α} {y : filter β} (i :
   tendsto f (x i) y → tendsto f (⨅i, x i) y :=
 tendsto_le_left (infi_le _ _)
 
+lemma tendsto_sup {f : α → β} {x₁ x₂ : filter α} {y : filter β} :
+  tendsto f (x₁ ⊔ x₂) y ↔ tendsto f x₁ y ∧ tendsto f x₂ y :=
+by simp only [tendsto, map_sup, sup_le_iff]
```

Comment:
`tendsto_sup_left`? Please add `tendsto.sup_left : tendsto f x₁ y → tendsto f x₂ y → tendsto f (x₁ ⊔ x₂) y`.

Reply:
Wouldn't `tendsto_sup_left` be inconsistent with `tendsto_inf_left` and `tendsto_inf_right` where `left` and `right have another meaning ?

Reply:
Maybe `tendsto_left_sup` ?

Reply:
I feel that `tendsto_inf_left` should be called `tendsto.inf_of_left` and there should be `tendsto.inf_right : tendsto f x y₁ → tendsto f x y₂ → tendsto x (y₁ ⊓ y₂)`. Probably this means that we should postpone this decision to another PR.

Reply:
Are you okay if I keep `tendsto_sup` and `tendsto.sup` for now, so we can fix everything at once ?

Reply:
Yes.